assets/sass/_utils.scss

```diff
@@ -1,18 +1,41 @@
-// classes to show/hide elements on desktop/mobile
+/* ==========================================================================
+   Utilities
+   ========================================================================== */
+/* Low level utility classes and display helpers */
+
+@include mq('desktop', 'max') {
+    .u-wide-only {
+        display: none !important;
+    }
+}
+
+@include mq('desktop') {
+    .u-not-wide {
+        display: none !important;
+    }
+}
 
 @include mq('tablet', 'max') {
-    .desktop-only { display: none !important; }
+    .u-desktop-only,
+    .desktop-only {
+        display: none !important;
+    }
 }
 
 @include mq('tablet') {
-    .mobile-only { display: none !important; }
+    .u-mobile-only,
+    .mobile-only {
+        display: none !important;
+    }
 }
 
+.u-hidden,
 .hidden {
     display: none;
 }
 
 // via http://simplyaccessible.com/article/three-pitfalls-text-alternatives/
+.u-visually-hidden,
 .visually-hidden {
```

Comment:
I think rather than supporting both naming conventions, let's just standardise on one of them (eg. the `.u-` prefix version) – same for the `.mobile-only` classes above this. I'm happy to do the search/replace job! Think it'd be nice to set a precedent here as we don't have any legacy templates to support or anything.

Reply:
Sure thing, that works for me if you're happy to standardise on this prefix for utility classes.